### PR TITLE
Ruslan us 015 test configure page of folder

### PR DESCRIPTION
**Pull request by RusAli**

### CODE REVIEW

File: src/test/java/US_15_FolderTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import runner.BaseTest;
+
+public class US_15_FolderTest extends BaseTest {
```
Comment (ViktorPodgornov) [inline src/test/java/US_15_FolderTest.java:7]: It's better not to use an underscore in the class name.

File: src/test/java/US_15_FolderTest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    String actualURL = getDriver().getCurrentUrl();
+
+    Assert.assertEquals(actualURL,expectedUrl);
+
+    getDriver().findElement(By.id("jenkins-home-link")).click();
+    getDriver().findElement(By.xpath("//a[contains(text(),'Configure')]")).click();
+    getDriver().findElement(By.linkText("Delete Folder")).click();
+    getDriver().findElement(By.id("yui-gen1-button")).click();
+
```
Comment (ViktorPodgornov) [inline src/test/java/US_15_FolderTest.java:28]: Can be deleted.

File: src/test/java/US_15_FolderTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    Thread.sleep(1000);
+
+    String actualURL = getDriver().getCurrentUrl();
+
+    Assert.assertEquals(actualURL,expectedUrl);
+
+    getDriver().findElement(By.id("jenkins-home-link")).click();
+    getDriver().findElement(By.xpath("//a[contains(text(),'Configure')]")).click();
+    getDriver().findElement(By.linkText("Delete Folder")).click();
+    getDriver().findElement(By.id("yui-gen1-button")).click();
```
Comment (ViktorPodgornov) [inline src/test/java/US_15_FolderTest.java:27]: You complete some actions after the Assert without adding a new Assert. Asserts should preferably be at the end of the code.

**Comment by ViktorPodgornov** [comment]
Make a link in Trello for PR.

File: src/test/java/US_15_FolderTest.java (hunk 1 above)
Comment (RusAli) [reply src/test/java/US_15_FolderTest.java:7]: Thanks. I will refactor class name after video lesson where we find out how to name,connect trello and etc.

File: src/test/java/US_15_FolderTest.java (hunk 3 above)
Comment (RusAli) [reply src/test/java/US_15_FolderTest.java:27]: This actions need to delete created folder and clean directory for recycling test

File: src/test/java/US_15_FolderTest.java (hunk 1 above)
Comment (ViktorPodgornov) [reply src/test/java/US_15_FolderTest.java]: It is not a strict rule)). But even Intellij "swears" at it.

File: src/test/java/US_15_FolderTest.java (hunk 3 above)
Comment (ViktorPodgornov) [reply src/test/java/US_15_FolderTest.java]: So the test is built incorrectly, nothing should be written after Asserts. Irina spoke about it.

**Comment by RusAli** [comment]
[TC_015.001 | Test configure page of folder](https://trello.com/c/qs0gclsR/112-tc015001-test-configure-page-of-folder)

File: src/test/java/US_15_FolderTest.java (hunk 1 above)
Comment (RusAli) [reply src/test/java/US_15_FolderTest.java]: Thanks) refactored

File: src/test/java/US_15_FolderTest.java (hunk 2 above)
Comment (RusAli) [reply src/test/java/US_15_FolderTest.java]: refactored

File: src/test/java/US_15_FolderTest.java (hunk 3 above)
Comment (RusAli) [reply src/test/java/US_15_FolderTest.java]: refactored and created separated methods

**Comment by ViktorPodgornov** [comment]
In fact, the user story itself is written incorrectly. Creating a folder is one functionality, and the configure page is another functionality. Nothing is said about checking the deletion of a folder. The test case also says nothing about checking for folder deletion. All these three functionals are separate three test cases, and they are interdependent.

**Review by ViktorPodgornov — APPROVED** [review]

**Review by EugeniaSrednitskaya — APPROVED** [review]

**Pull request merged**